src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -739,4 +747,33 @@ export default class DefaultAudioVideoController implements AudioVideoController
       this.meetingSessionContext.signalingClient.resume([streamId]);
     }
   }
+
+  sendEvent(name: string, attribute: { [attributeName: string]: string | string[] }): void {
+    this.forEachObserver((observer: AudioVideoObserver) => {
+      Maybe.of(observer.eventDidReceive).map(f => f.bind(observer)(name, attribute));
+    });
+  }
+
+  getAttributes(
+    browserBehavior: BrowserBehavior = this.meetingSessionContext.browserBehavior,
```

Comment:
We instantiate a new   `this.meetingSessionContext = new AudioVideoControllerState();` in actionConnect, so I will add a DefaultBrowserBehavior instance to the meetingSessionContext in the constructor, but it will be overridden in actionConnect, which is what we do for other meetingSessionContext properties also, like the logger.